Packs/RiskSense/Integrations/RiskSense/RiskSense.yml

```diff
@@ -47,21 +47,21 @@ script:
             - Host Name
             - IP Address
             - Criticality
-          description: The RiskSense host attribute by which to filter the results. Can be 'Host Name', 'IP Address', or 'Criticality'. Apart from the available choices you can provide the attributes supported by RiskSense API. Refer to the API /host/filter API to get the list of supported attributes. The uid of filter attributes must be provided here, for example assessment_labels, asset_tags, cvss3BaseI, etc. If specified, 'value' argument is mandatory.
+          description: The RiskSense host attribute by which to filter the results. Can be 'Host Name', 'IP Address', or 'Criticality'. Apart from the available choices, you can provide the attributes supported by RiskSense API. Refer to the API /host/filter API to get the list of supported attributes. The uid of filter attributes must be provided here, for example, assessment_labels, asset_tags, cvss3BaseI, etc. If specified, 'value' argument is mandatory.
         - name: operator
           auto: PREDEFINED
           predefined:
             - EXACT
             - IN
             - LIKE
             - BETWEEN
-          description: The match operator that should be applied for filtering the hosts based on 'fieldname' and 'value'. Can be 'EXACT' - filter records exactly matching the criteria; 'IN' - filter records matching any one of the comma-separated values; 'LIKE' - filter records with value matching the the specified pattern. All the records fieldname value contains the string provided in value; 'BETWEEN' - filter the records with fieldname value falling in the numerical/date range provided. This argument also accepts other operator values supported by the RiskSense API. Refer the API documentation for the list of supported operators.
-        - name: exclusive_operator
+          description: The match operator should be applied for filtering the hosts based on 'fieldname' and 'value'. Can be 'EXACT' - filter records exactly matching the criteria; 'IN' - filter records matching any one of the comma-separated values; 'LIKE' - filter records with the value matching the specified pattern. All the records fieldname value contains the string provided in value; 'BETWEEN' - filter the records with fieldname value falling in the numerical/date range provided. This argument also accepts other operator values supported by the RiskSense API. Refer to the API documentation for the list of supported operators.
+        - name: exclusivity
```

Comment:
This will break backward compatibility for existing users.
changing argument names, parameters names, command names, outputs, all breaks backward compatibility. 
There are 3 options:
1. Not break backward compatibility - just use the same old names and in the code handle it. - which is seems the best option here
2. Deprecate arguments (just add `deprecate: true` under that specific arg), add new one, but you have still have to support the old one
3. Create new integration and call it `v2`, `v3`, etc

Reply:
we got an update from client (Hitesh) that we should not be facing any backward compatibility issue as currently given integration is not used by customers. So this is the time was can make this change.